data/database/src/commonMain/kotlin/com/tunjid/heron/data/database/migrations/Migration28To29PostBookmarkViewingProfileId.kt

```diff
@@ -0,0 +1,38 @@
+package com.tunjid.heron.data.database.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.SQLiteConnection
+import androidx.sqlite.execSQL
+import com.tunjid.heron.data.core.models.Constants
+
+internal object Migration28To29PostBookmarkViewingProfileId : Migration(28, 29) {
+    override fun migrate(connection: SQLiteConnection) {
+        connection.execSQL(
+            """
+            CREATE TABLE postBookmark_new (
+                viewingProfileId TEXT NOT NULL DEFAULT '${Constants.UNKNOWN}',
+                postUri TEXT NOT NULL,
+                postId TEXT NOT NULL,
+                createdAt INTEGER NOT NULL,
+                PRIMARY KEY(viewingProfileId, postUri),
+                FOREIGN KEY(postUri) REFERENCES posts(uri) ON DELETE CASCADE
+            )
+            """.trimIndent(),
+        )
+
+        connection.execSQL(
```

Comment:
Since this entity has not been saved before, you don't need to actually migrate anything.